docs/modules/execa.md

```diff
@@ -0,0 +1,23 @@
+# Execa
+
+Execa is process execution library that simplifies command execution by providing programmatic API on top of `node:child_process`.
```

Comment:
this isn't really consistent with the rest of the repo. we don't need to explain what `execa` does, rather we just need to mention why you should move off it

in this case, you'd move off it because there are lighter alternatives when you don't need extended features

otherwise we're just duplicating info you can get from execa's own readme

Reply:
Valid point, should I remove the list of important features too? Personally, I want to add it since users might not know what those 'kitchen-sink' features are.

Reply:
i would just mention that if you need templating or other high level features, you might not want to switch. soon as we start listing any features though, it'll lead us down a rabbit hole and we'll end up replicating execa's readme again

remember its the job of the manifest/tooling to suggest this replacement under the right circumstances. once we have a usage-based lint rule and codemod, this'll only be suggested when the user doesn't use any of those extended features

so we don't need to go into much depth here